addon/components/upf-editor/component.js

```diff
@@ -8,30 +8,45 @@ export default Component.extend({
   layout,
 
   summernoteContext: null,
+  activatedCustomButtons: 'VideoUploadButton,PDFUploadButton',
   customButtons: [],
 
   hideVideoUploadModal: true,
+  hidePDFUploadModal: true,
 
   init() {
     let self = this;
 
-    let VideoUploadButton = (context) => {
+    this.VideoUploadButton = (context) => {
```

Comment:
why not use a map like `uploadBtn['video'] ` ? and rename `VideoUploadButton` and `PDFUploadButton` by `video,pdf` ?